tensorflow/core/ops/string_ops.cc

```diff
@@ -203,6 +203,7 @@ REGISTER_OP("StringSplit")
     .Output("values: string")
     .Output("shape: int64")
     .Attr("skip_empty: bool = true")
+    .Attr("encoding: string = ''")
```

Comment:
Please also include a description of what this is in the documentation string below

Reply:
Thanks @asimshankar. The docs has been updated.